vignettes/synapser.Rmd

```diff
@@ -470,8 +470,10 @@ synGetPermissions(project, principalId = 273950)
 ```
 The first time an entity is shared, an ACL object is created for that entity. Let's make project public:
 ```{r collapse = TRUE}
-acl <- synSetPermissions(project, principalId = 273950, accessType = list("READ"))
-acl
+# these calls fail in DEV "Cannot assign permissions to anonymous. To share resources with anonymous users, use the PUBLIC group id (273949)"
```

Comment:
the fix is simply to update the principalId in the test to be the reserved ID for 'public'.